service/templates/task.html

```diff
@@ -242,6 +242,10 @@ <h1 class="one-liner" id="task-title">
                       <td class="icon"><i class="fa fa-calendar"></i></td>
                       <td>Finished on [[task.finish_time | dateFromNanoseconds | convertDateToUserTimezone:userTz:"MMM D, YYYY h:mm:ss a"]]</td>
                     </tr>
+                    <tr ng-show="task.task_end_details.status == 'failed' && (task.task_end_details.type == 'setup' || task.task_end_details.type == 'system') ">
+                      <td class="icon"><i class="fa fa-code"</i></td>
```

Comment:
The problem is that the first <i element doesn't have a matching >

Reply:
oh yikes sorry